viewing_party/main.py

```diff
@@ -0,0 +1,249 @@
+def create_movie(title, genre, rating):
+    """
+    Creates a dictionary representing the description of a movie
+    """
+    movie_dict = {
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+
+    if movie_dict["title"] is None or movie_dict["genre"] is None or\
+            movie_dict["rating"] is None:
```

Comment:
I wonder if the slash is a typo? I removed it locally so it still passed that test, that's peculiar. Other than that this looks great!